src/main/java/com/getify/minify/Minify.java

```diff
@@ -0,0 +1,103 @@
+//
+//  Minify.java
+//  jsonminifier - Ported from https://github.com/getify/JSON.minify
+//
+//  Created by Bernhard Gass on 8/01/13.
+//  Copyright © 2013 Bernhard Gass. All rights reserved.
+
+package com.getify.minify;
```

Comment:
@getify: yup, `io.github.getify`would be a valid package prefix and maven `groupId`. It's fairly common, especially among organizations/groups who are committed to github or are on a razor budget that doesn't include domain names.